content/blog/2022-05-16-may-22-heartbeat.md

```diff
@@ -0,0 +1,395 @@
+
+https://media.giphy.com/media/MrCYIN3x0SgdG/giphy.gif
+
+## 🚀~~Mission Impossible~~ - We have a mission statement!
+
+We did it! This year we surveyed the entire team to arrive at a mission
+statement for Iterative. It was no small feat to decide on what it should be
+given the early stage of our industry, the variety of our tools, and always a
+struggle - figuring out the best and most concise way to convey these ideas (you
+know our penchant for abbreviations). But we persevered and succeeded. Behold
+Iterative's new mission statement:
+
+> We deliver the best developer experience for machine learning teams by
+> creating an ecosystem of open, modular ML tools.
+
+As always the door is open for your feedback on how we can serve your needs
+better!
+
+## ODSC East
+
+We attended our first post-pandemic, in-person conference in Boston last month.
+It was awesome to be together as a team, see
+[**Dmitry Petrov**](https://twitter.com/FullStackML),
+[**Milicia McGregor**](https://twitter.com/FlippedCoding), and
+[**Alex Kim**](https://www.linkedin.com/in/alex000kim/) in action, and talk to
+attendees and other vendors at the conference. We are looking forward to
+[MLOps World](https://mlopsworld.com/) next month!
+
+![Iterative Team at ODSC East](/uploads/images/2022-05-16/odsc.jpeg '=800')
+_Iterative team (left to right) - Mike Moynihan, me, Dave Berenbaum, Daniel
+Barnes, (DeeVee), Rob De Wit, Milicia McGregor, Dmitry Petrov, Jervis Hui, Alex
+Kim, Chaz Black_
+
+## ✍🏼 Tons of new content on the blog
+
+Our team has been on fire creating content for you. 🔥 Don't miss the following:
+
+- Needing to get started with CML and AWS?
+  [**Rob de Wit**](https://www.linkedin.com/in/rcdewit?miniProfileUrn=urn%3Ali%3Afs_miniProfile%3AACoAAA5CEPkB9fI02IpClBKhRdq2brULPHMhmR8&lipi=urn%3Ali%3Apage%3Ad_flagship3_search_srp_all%3B9MrcxBhQSG6IKzSgJDyfQQ%3D%3D)
+  shows you how to train and save your models with CML in a two-part series
+  using a
+  [self-hosted AWS EC2 runner](https://dvc.org/blog/CML-runners-saving-models-1)
+  and
+  [with CML and DVC on a dedicated AWS EC2 runner](https://dvc.org/blog/CML-runners-saving-models-2)
+- The
+  [Part 1](https://dvc.org/blog/end-to-end-computer-vision-api-part-1-data-versioning-and-ml-pipelines),
+  [Part 2](https://dvc.org/blog/end-to-end-computer-vision-api-part-2-local-experiments)
+  and
+  [Part 3](https://dvc.org/blog/end-to-end-computer-vision-api-part-3-remote-exp-ci-cd)
+  tutorials of [**Alex Kim's**](https://www.linkedin.com/in/alex000kim/)
+  End-to-End Computer Vision API project are out and filled with great learning!
+- [**Milecia McGregor**](https://twitter.com/FlippedCoding) brings the monthly
+  roundup of the Community's best technical questions in our latest
+  [Community Gems.](https://dvc.org/blog/april-22-community-gems) 💎
+
+## ✨ Shiny New Docs
+
+We have a [new doc page](https://dvc.org/doc/start/visualization) showcasing the
+new visualizations added to the
```

Comment:
This page is under an `experiments` subdirectory, i.e. https://dvc.org/doc/start/experiments/visualization

Also, the link would ideally be root-relative so that the advanced local link handling can be used, i.e. `/doc/start/experiments/visualization`